packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDepositRow.tsx

```diff
@@ -261,55 +274,80 @@ export function TransactionsTableDepositRow({
     () =>
       sanitizeTokenSymbol(tx.asset, {
         erc20L1Address: tx.tokenAddress,
-        chain: l1.network
+        chain: getWagmiChain(tx.parentChainId)
       }),
-    [l1.network, tx.asset, tx.tokenAddress]
+    [tx.parentChainId, tx.asset, tx.tokenAddress]
   )
 
   const customAddressTxPadding = useMemo(
     () => (isCustomDestinationAddressTx(tx) ? 'pb-11' : ''),
     [tx]
   )
 
+  const rowHeight = useMemo(() => {
+    if (isPending(tx) || isFailed(tx)) {
+      return isCustomDestinationAddressTx(tx) ? 'h-[126px]' : 'h-[94px]'
+    }
+    return isCustomDestinationAddressTx(tx) ? 'h-[117px]' : 'h-[85px]'
+  }, [tx])
+
   if (!tx.sender || !address) {
     return null
   }
 
   return (
-    <tr
+    <div
+      data-testid={`deposit-row-${tx.txId}`}
       className={twMerge(
-        'relative text-sm text-dark',
-        bgClassName || 'bg-cyan even:bg-white',
+        'relative grid h-full grid-cols-[150px_250px_140px_280px_150px] border-b border-dark text-sm text-dark',
+        bgClassName,
         className
       )}
-      data-testid={`deposit-row-${tx.txId}`}
+      style={{ gridAutoRows: rowHeight }}
     >
-      <td className={twMerge('w-1/5 py-3 pl-6 pr-3', customAddressTxPadding)}>
+      <div
+        className={twMerge(
+          'pl-6 pr-3 align-middle',
+          customAddressTxPadding,
+          isTxCompleted(tx) ? 'py-3' : 'py-4'
```

Comment:
With reskin we won't need those because we will consolidate deposits and withdrawals with a single component. For now we manually adjust rows so they look right